src/app/messages.ts

```diff
@@ -38,12 +60,39 @@ export const errMessages = {
     }
   },
   Login: {
-    
+    "login": "Oops... Invalid email or password, please type it again."
+  },
+  PasswordValidation: {
+    mismatch: {
+      "mismatch": "Password mismatch ..."
+    },
+    minlength: {
+      "minlength": "Password minimum length is 8 characters ..."
+    }
   },
   Registration: {
-
+    mismatch: {
+      "match": "The passwords entered do not match."
+    },
+    error: {
+      "error": "Sorry, something went wrong. Please contact " // after the "contact", the content will be a variable, please check with Registration/model.ts file to view detail
+    },
+    invalidUser: {
+      "account": "Whoops... Account not found. Please contact " // after the "contact", the content will be a variable, please check with Registration/model.ts file to view detail
+    },
+    noPassword: {
+      "password": "Whoopes... Unable to register, invalid password."
```

Comment:
Whoops... Sorry, we have been unable to register you. You must enter a valid password.